scripts/ci-deb-build-test

```diff
@@ -19,6 +21,21 @@ apt-get install -y \
     cryptsetup \
     tzdata \
     curl wget git
+
+if [ $OS_MAJOR -gt 23 ];  then 
+# install dependencies for 24.04 (ubuntu 24.04 does not have python2)
+apt-get install -y \
+    devscripts \
+    debhelper \
+    dh-autoreconf \
+    help2man \
+    libarchive-dev \
+    libssl-dev \
+    python3 \
+    uuid-dev \
+    golang-go \
+    dh-apparmor
```

Comment:
dh-apparmor should also be added to the list of packages needed in INSTALL.md when compiling for Debian, not just in the ci script.